config/i18n-tasks.yml

```diff
@@ -0,0 +1,170 @@
+<% require './lib/i18n_tasks/coffee_erb_locale_matcher.rb' %>
+
+# i18n_tasks finds and manages missing and unused translations: https://github.com/glebm/i18n-tasks
+
+# The "main" locale.
+base_locale: en
+## All available locales are inferred from the data by default. Alternatively, specify them explicitly:
+# locales: [es, fr]
+## Reporting locale, default: en. Available: en, ru.
+# internal_locale: en
+
+# Read and write translations.
+data:
+  ## Translations are read from the file system. Supported format: YAML, JSON.
+  ## Provide a custom adapter:
+  # adapter: I18n::Tasks::Data::FileSystem
+
+  # Locale files or `Find.find` patterns where translations are read from:
+  read:
+    ## Default:
+    # - config/locales/%{locale}.yml
+    ## More files:
+    # - config/locales/**/*.%{locale}.yml
+    - config/locales/%{locale}/*.yml
+    - config/locales/%{locale}/controllers/*.yml
+    - config/locales/%{locale}/views/*.yml
+    - config/locales/%{locale}/external/*.yml
+
+  # Locale files to write new keys to, based on a list of key pattern => file rules. Matched from top to bottom:
+  # `i18n_tasks normalize -p` will force move the keys according to these rules
+  write:
+    - ['{devise, simple_form}.*', 'config/locales/%{locale}/external/\1.yml']
+    - ['common.*', 'config/locales/%{locale}/common.yml']
+    - ['javascripts.*', 'config/locales/%{locale}/javascripts.yml']
+    - ['activerecord.*', 'config/locales/%{locale}/models.yml']
+    - ['{:}.model.*', 'config/locales/%{locale}/models.yml']
+    - ['{:}.controller.*', 'config/locales/%{locale}/controllers/\1.yml']
+    - ['{:}.{*_service}.*', 'config/locales/%{locale}/controllers/\1.yml']
+    - ['{:}.*', 'config/locales/%{locale}/views/\1.yml']
+    ## Catch-all default:
+    - config/locales/%{locale}/unsorted.yml
+
+  # External locale data (e.g. gems).
+  # This data is not considered unused and is never written to.
+  external:
+    ## Example (replace %#= with %=):
+    # - "<%#= %x[bundle info vagrant --path].chomp %>/templates/locales/%{locale}.yml"
+
+  ## Specify the router (see Readme for details). Valid values: conservative_router, pattern_router, or a custom class.
+  # router: conservative_router
+
+  yaml:
+    write:
+      # do not wrap lines at 80 characters
+      line_width: -1
+
+  ## Pretty-print JSON:
+  # json:
+  #   write:
+  #     indent: '  '
+  #     space: ' '
+  #     object_nl: "\n"
+  #     array_nl: "\n"
+
+# Find translate calls
+search:
+  ## Paths or `Find.find` patterns to search in:
+  paths:
+    - app/
+    - spec/
+    - app/assets/javascripts/
+
+  ## Root directories for relative keys resolution.
+  # relative_roots:
+  #   - app/controllers
+  #   - app/helpers
+  #   - app/mailers
+  #   - app/presenters
+  #   - app/views
+
+  ## Directories where method names which should not be part of a relative key resolution.
```

Comment:
Since we are now allowing the relative key resolution, we can also instruct Rubocop to watch out for potential violations. To do so, modify the `.rubocop/rails.yml` and change the `Rails/I18nLazyLookup` setting.